src/Components/CarouselItem.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { LandingIcon } from './LandingIcon';
+
+function CarouselItem({ name, company, profilePic, comment, className }) {
+  return (
+    <div className={`bg-gradient-to-t from-[#5A98F2] to-[#67C3F3] rounded-3xl px-[9vw] py-[3vw] mx-[3vw] ${className} overflow-hidden flex flex-col`}>
```

Comment:
this could have the transition property right? where are we applying it for the carrousel?